src/main/java/com/flowingcode/vaadin/addons/twincolgrid/TwinColGrid.java

```diff
@@ -105,12 +106,10 @@
 
 	private final VerticalLayout buttonContainer;
 
-	private Grid<T> draggedGrid;
+	private TwinColModel<T> draggedGrid;

```

Comment:
```suggestion
	private Grid<T> draggedGrid;

```
(i.e. no change here)